Zend/tests/object_array_cast.phpt

```diff
@@ -14,11 +14,11 @@ var_dump($obj);
 ?>
 --EXPECT--
 object(stdClass)#1 (3) {
-  [0]=>
```

Comment:
Possibly a BC break here, although it shouldn't change much for consumers, unless they rely on `array_keys()`